tests/test_cli.py

```diff
@@ -0,0 +1,79 @@
+import imp
+import shlex
+import sys
+import threading
+
+import runs
+
+import pytest
+from strip_ansi import strip_ansi
+
+
+def test_cli_version(capfd):
+    """
+    Verify `mqttshark --version` succeeds.
+    """
+    run_mqttshark("--version")
+    out, err = capfd.readouterr()
+    assert out.strip() == "mqttshark version 0.0.1"
+
+
+def test_cli_capture(capfd, mosquitto):
+    """
+    Verify capturing a single MQTT PUBLISH succeeds.
+    """
+    interface_name = "lo"
+    if sys.platform == "darwin":
+        interface_name = "lo0"
+
+    ev = threading.Event()
+    def mqttshark():
+        run_mqttshark(f"-i {interface_name}")
+    t = threading.Thread(target=mqttshark)
+    t.start()
+    ev.wait(0.5)
+
+    command = """mosquitto_pub -t 'testdrive' -m '{"temperature": 42.42}'"""
+    runs.run(command)
+    ev.wait(0.25)
+
+    runs.run("sudo pkill tshark")
+    ev.wait(0.25)
+
+    out, err = capfd.readouterr()
+
+    assert out is not None, "Captured output was expected, but it is empty"
+    out = strip_ansi(out)
+    assert 'CONNECT     clientid=""  keepalive=60  cleansession=True  protover=4' in out
+    assert 'CONNACK     rc=0 (accepted)' in out
+    assert 'PUBLISH(0)  retain=0  topic="testdrive"  payload="{"temperature": 42.42}"' in out
+    assert 'DISCONNECT' in out
+
+
+def test_cli_load_module(capsys):
+    """
+    Basic test function, loading `mqttshark` as Python module.
+    This is too verbose!
+
+    :param capsys:
+    :return:
+    """
+    command = "mqttshark --version"
+    sys.argv[1:] = shlex.split(command)
+    with pytest.raises(SystemExit):
+        imp.load_source(name="mqttshark", pathname="mqttshark")
+    out, err = capsys.readouterr()
+    assert out.strip() == "mqttshark version 0.0.1"
```

Comment:
Just a basic example how to load the current program as a module. This test case will also probably evolve in the future.